addons/website_project_issue/controllers/main.py

```diff
@@ -37,6 +37,12 @@ def my_issues(self, page=1, date_begin=None, date_end=None, sortby=None, filterb
         searchbar_filters = {
             'all': {'label': _('All'), 'domain': []},
         }
+        searchbar_inputs = {
```

Comment:
If you reallyw ant to customize it, you can simply override in template the `searchbar_inputs[input]`